Development

DilushaLakshan opened the pull request:
Modify the display available packages module for fetch real data from the database.
![Tour-packages](https://github.com/user-attachments/assets/8c64bc6c-dda2-4f9f-a26d-9311f4be8db3)

Modify the package-details module for fetch real data from the database.
![Package-Details (1)](https://github.com/user-attachments/assets/7bfa778d-0562-45d1-9436-557b6d7d7a22)

Partially completed the tour planning module.
![Plan-a-Tour](https://github.com/user-attachments/assets/13db28ce-2144-4464-a94e-1c6c1c6334ca)

Completed the update tour package functionality.
![Update-Tour-Package](https://github.com/user-attachments/assets/1c6629b7-84f8-46c7-b449-14267450f3af)



Also modify the add-destination feature for send packing list item ids to the database.
Modify the all available feature for working with sessions and cookies.

DilushaLakshan committed:
change index file and connection file.

DilushaLakshan committed:
update home page main header

DilushaLakshan committed:
made layout designing parts

DilushaLakshan committed:
Fixed the issues in code lines.

DilushaLakshan committed:
Fixed issues in code lines.

DilushaLakshan committed:
Following modules was implemented.
Hotel registration.
Vehicle Registration.
Adding tour destinations.

Also following pages were updated.
destination-details.php
destinations.php
loginProcess.php
allStaffMembers.php

DilushaLakshan committed:
Completed the add tour package functionality

DilushaLakshan committed:
Made chages.

DilushaLakshan committed:
Convert hardcode data into fetch from the database

DilushaLakshan committed:
Completed update tour package and modified tour planning process and add-destination process for storing the packing list ids.

dulanjayabhanu commented on add-hotel.php:
```diff
@@ -0,0 +1,128 @@
+                                <label>No. of Rooms</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="h-rooms">
```
Add `min`, `max` attributes to the input element.

DilushaLakshan replied:
OK

dulanjayabhanu commented on add-hotel.php:
```diff
@@ -0,0 +1,128 @@
+                                <label>Price Per. Room (LKR)</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="h-price">
```
If you have to use an input element for `price`. You can use an attribute called `pattern`. It helps to prevent invalid value entered by the user. 

Refer [this](https://stackoverflow.com/questions/29603080/html5-input-type-number-with-regex-pattern)

DilushaLakshan replied:
Got it

dulanjayabhanu commented on add-tourpackage.php:
```diff
@@ -0,0 +1,483 @@
+                                <label>Price</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="price">
```
If you have to use an input element for `price`. You can use an attribute called `pattern`. It helps to prevent invalid value entered by the user.

Refer [this](https://stackoverflow.com/questions/29603080/html5-input-type-number-with-regex-pattern)

DilushaLakshan replied:
Got it

dulanjayabhanu commented on add-tourpackage.php:
```diff
@@ -0,0 +1,483 @@
+                                <label>Number of Vehicles</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="no-of-vehicles">
```
Add `min`, `max` attributes to the input element.

DilushaLakshan replied:
Ok

dulanjayabhanu commented on index.php:
```diff
@@ -0,0 +1,111 @@
+            <?php
+            if (isset($_SESSION["user"])) {
+                $uID = $_SESSION["user"]->id;
+                include 'navbar-logged-in.php';
```
Don't use 2 files (`navbar.php` and `navber-logged-in.php`) for navbar. You can manage this feature using only 1 file. You must program the navbar (use session management to validate the user in the `navbar.php` file) and include this file in `index.php`.

DilushaLakshan replied:
OK

dulanjayabhanu commented on one-member.php:
```diff
@@ -0,0 +1,99 @@
+    <div class="container-fluid">
+        <div class="row">
+            <?php
+            $memberID = $_GET["memberID"];
```
You must check if the parameter exists before assigning to the variable.

DilushaLakshan replied:
OK

dulanjayabhanu reviewed (approved):
Looks good, ready to merge!

DilushaLakshan committed:
Mainly completed the following modules.
Tour package filtering
Send emails when signing in
Bookings and booking confirmation

Modify the following modules
Add tour package
Update tour packages
Add destinations
Update destinations
Add vehicle
Update vehicles

Also implement the footer and a part of visual dashboard using static data and Chart.js library.

DilushaLakshan committed:
Mainly completed the following modules.
Tour package filtering
Send emails when signing in
Bookings and booking confirmation

Modify the following modules
Add tour package
Update tour packages
Add destinations
Update destinations
Add vehicle
Update vehicles

Also implement the footer and a part of visual dashboard using static data and Chart.js library.

DilushaLakshan committed:
Concluded merge and resolved conflicts
